bundled/src/androidMain/kotlin/DatadogInitializer.kt

```diff
@@ -50,6 +50,7 @@ private fun Site.toDatadogType(): DatadogSite = when (this) {
     Site.US1 -> DatadogSite.US1
     Site.US3 -> DatadogSite.US3
     Site.US5 -> DatadogSite.US5
-    Site.EU1 -> DatadogSite.EU1
+    Site.EU -> DatadogSite.EU1
+    Site.AP1 -> DatadogSite.AP1
```

Comment:
ultra nit: on iOS and JS, these are listed in reverse order. Would probably be good to be consistent.

```suggestion
    Site.AP1 -> DatadogSite.AP1
    Site.EU -> DatadogSite.EU1
```